src/module_PI/Raspberry_PI/JUnitTests/EmailNotifierTest.java

```diff
@@ -0,0 +1,77 @@
+package module_PI.Raspberry_PI.JUnitTests;
+
+import static org.junit.Assert.*;
+
+import org.junit.*;
+import org.junit.rules.*;
+
+import module_PI.Raspberry_PI.main.EmailNotifier;
+
+public class EmailNotifierTest {
+	@Rule
+	public ExpectedException expectedException = ExpectedException.none();
+	
+	public static EmailNotifier testEmailNotifier = new EmailNotifier("testAddress@email.com", "testMessage");
+
+	@Test
+	public void testEmailNotifier_EmailAddressAndMessage() {
+		EmailNotifier emailNotifier = new EmailNotifier("test@test.com", "test");
+		assertEquals(emailNotifier.getAttachmentFilePath(), null);
+	}
+	
+	@Test
+	public void testEmailNotifier_EmailAddressMessageAndAttachment(){
+		EmailNotifier emailNotifier = new EmailNotifier("test@test.com", "test", "README.md");
+		assertEquals(emailNotifier.getAttachmentFilePath(), "README.md");
+	}
+	
+	@SuppressWarnings("unused")
+	@Test
+	public void testEmailNotifier_AttachmentDoesntExist(){
+		expectedException.expectMessage("File nonexistantfile.txt can't be found");
```

Comment:
[Expected Exception](http://junit.org/junit4/javadoc/4.12/org/junit/rules/ExpectedException.html) is basically a unit test that expects a certain exception from the following code. In this case, nonexistantfile.txt doesn't exist, so we expect it to throw an exception that gives that particular message.